resume_pj/index.html

```diff
@@ -0,0 +1,148 @@
+                <div class="navbar-nav">
+                  <a class="nav-link active" aria-current="page" href="#">Home</a>
+                  <a class="nav-link" href="#">Features</a>
+                  <a class="nav-link" href="#">Pricing</a>
+                  <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>
+                </div>
+              </div>
+            </div>
+            </div>
+          </nav>
+    </header>
+
+
+    <main>
+        <div class="border border-warning container-fluid">
+            <div class="row border border-primary ">
+                <div class="col-10 offset-1 ">
+                    <div class="row justify-content-center text-center background-beige">
+               
+                    <img src="./img/photo-profile.jpeg" class="rounded-circle col-5" alt="profile pic">
+                    <h2>Maya Nelson</h2>
+                    <hr>
+                   
+                        <div class="row justify-content-around">
+                            
+                                <button type="button" class="col-5 btn btn-outline-primary">RESUME</button>
+                            
+                            
+                                <button type="button" class="col-5 btn btn-outline-primary">PROJECTS</button>
+                        </div>
+                        </div>
+                    <div class="row social-class justify-content-center">
+                        <div class="col-1">
+                            <a href="">
+                            <i class="fa fa-instagram"></i>
+                            </a>
+                        </div>
+                        <div class="col-1">
+                            <a href="">
+                                <i class="fa fa-facebook"></i>
+                            </a>
+                        </div>
+                        <div class="col-1">
+                            <a href="">
+                                <i class="fa fa-twitter" style="color: rgb(32, 32, 32);"></i> 
+                            </a>
+                        </div>
+                        <div class="col-1">
+                            <a href="">
+                                <i class="fa fa-linkedin-square"></i>
+                            </a>
+                        </div>
+                    </div>
+                    
+                    </div>
+                </div>
+            </div>
+                 <div class="border border-danger col-md-7 col-12 hello-section">
+                    <h1>Hello</h1>
+                    <h3>Here's who I am & what I do</h3>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font.
+
+ 
+
+I’m a great place for you to tell a story and let your users know a little more about you.
```

Comment:
BEar in mind TML doesn't take line breaks like this into account. If you want to split the 2 parts of the paragraph, use separate `p` tags for each